### PR TITLE
Observable lb

### PR DESCRIPTION
**Pull request by KaiSzuttor**
Description of changes:
 - contains #1594
 - adds an observable and test for the LB fluid velocity in polar coordinates at particle positions

PR Checklist
------------
 - [x] Tests?
   - [x] Interface
   - [x] Core 
 - [x] Docs?

### CODE REVIEW

File: src/core/observables/CylindricalLBFluxDensityProfileAtParticlePositions.hpp [hunk 1]
```diff
@@ -0,0 +1,109 @@
+#ifndef OBSERVABLES_CYLINDRICALLBFLUXDENSITYPROFILEATPARTICLEPOSITIONS_HPP
+#define OBSERVABLES_CYLINDRICALLBFLUXDENSITYPROFILEATPARTICLEPOSITIONS_HPP
+
+#include "CylindricalProfileObservable.hpp"
+#include "utils.hpp"
+#include "utils/Histogram.hpp"
+#include "lbgpu.hpp"
+
+namespace Observables {
+class CylindricalLBFluxDensityProfileAtParticlePositions : public CylindricalProfileObservable {
```
Comment (fweik) [inline src/core/observables/CylindricalLBFluxDensityProfileAtParticlePositions.hpp]: Please move the implementation to a .cpp file. Then you don't have to drag in the other headers here, which are not actually part of the interface. That also decreases the build time.

File: src/core/utils/Histogram.hpp [hunk 2]
```diff
@@ -1,3 +1,5 @@
+#ifndef HISTOGRAM_HPP
```
Comment (fweik) [inline src/core/utils/Histogram.hpp]: Should correspond to the file path, e.g. CORE_UTILS_HISTOGRAM_HPP, so that there can not be collisions.

**Review by fweik — CHANGES_REQUESTED** [review]

**Comment by KaiSzuttor** [comment]
just found there is still sth wrong...

**Comment by KaiSzuttor** [comment]
Ready for review @fweik

**Review by fweik — APPROVED** [review]

**Pull request merged**